identity-iota/Cargo.toml

```
@@ -12,6 +12,7 @@ description = "An IOTA Tangle intergration for the identity-rs library."
 
 [dependencies]
 async-trait = { version = "0.1", default-features = false }
+brotli = "3.3.2"
```

nit: disable unused features (we don't need `ffi-api`).

```suggestion
brotli = { version = "3.3", default-features = false, features = ["std"] }
```